.github/workflows/release-drafter.yml

```diff
@@ -0,0 +1,40 @@
+name: Draft Release
+
+on:
+  push:
+    tags:
+      - "*"
```

Comment:
Will this automatically set the version to `2.0.1`?

Reply:
No, this is just a trigger. Any tag pushed to this repo will trigger this workflow